tests/bot/exts/info/test_information.py

```diff
@@ -347,7 +346,7 @@ async def test_create_user_embed_expanded_information_in_moderation_channels(
 
         self.assertEqual(
             textwrap.dedent(f"""
-                Created: {"1 year ago"}
+                Created: {"<t:1:R>"}
```

Comment:
When `<t:1:R>` is typed out in Discord, it seems that it converts to 52 years ago. Is that supposed to be correct here?

Reply:
Since the decoded timestamp is 1, yes. We used to patch the function that gets the formatted timestamp to return `1 year ago` but since it is gone, we can just assert that it constructed a valid timestamp.

Reply:
I see. Thank you for this information.